### PR TITLE
Jetpack Manage: Pricing page - styling adjustments to match design specifications

### PR DESCRIPTION
**Pull request by coder-karen**
Related to https://github.com/Automattic/jetpack-manage/issues/266

## Proposed Changes

* This PR includes several style related changes, as well as minor location adjustments. Specifically:
* All font sizes should match design specs
* Variant selection is above the price in both the VaultPress Backup and the Security cards.
* Prices will show 'US$' in front.
* FAQ section removed to match design specs
* Backup, Boost and Akismet product icons match currently existing icons, not design specs, as discussed here: pf36In-Gb-p2#comment-1243 
* The “per bundle per month” and “per license per month” text that shows for bundles greater than 1 will now show on a new line. See pf36In-Gb-p2#comment-1297 

Design specs are at pf36In-Gb-p2.

## Testing Instructions

* To test, check out this branch and open `http://jetpack.cloud.localhost:3000/manage/pricing`. 
* Make sure the above listed design changes (plus everything else on the page) matches the design specs
* Note that this PR is just to make design / layout tweaks on the `/manage/pricing` page itself, so doesn't consider any functionality issues such as where the buttons lead to as some issues are still being worked out there.

Screenshot showing the full page:

Screenshot 2024-02-02 at 09 44 49

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?lat

### CODE REVIEW

**Review by keoshi — APPROVED** [review]
First off, this looks incredible @coder-karen ! Thanks so much for all the energy here. Adding a few notes below.

#### Buying multiple licenses of Add-ons or Woo Extensions doesn't work

When trying to purchase 10 licenses of 10GB of VP Backup space, I notice we're passing the param `bundle_size=10`, but the review step only shows one.

image

This seems unrelated to this particular PR, but wanted to surface it ASAP. Perhaps this is related to https://github.com/Automattic/jetpack-manage/issues/230 ? Also related is this P2 thread: pf36In-DL-p2#comment-989

cc @simonktnga8c 

#### Mobile: license number selector

Noticed that on mobile the license number selector is limited in width, displaying all the options very tightly. Could we instead let the selector take however much horizontal space it needs and allow people to scroll horizontally to see more options?

Before | After
--|--
image | image

The other solution would be a dropdown, but showing only one value at a time does diminish the immediate value of Manage pricing.

#### Extra spacing in the pricing terms

image

#### Per vs `/`

Some items contain `per bundle...` while others using a forward slash `/month`. Can we standardize these descriptions so that they all use `per ...`?

---

Thanks again, @coder-karen. This is outstanding!

**Comment by coder-karen** [comment]
Thanks @keoshi! 

> When trying to purchase 10 licenses of 10GB of VP Backup space, I notice we're passing the param bundle_size=10, but the review step only shows one.

Yes, that's intentional at this stage (my reading of the issue you linked and the design wrap up is that we show the add-ons extensions for bundles but the bundle functionality itself doesn't work just yet).

That said - perhaps we should remove the param from being passed through for those for now to avoid confusion 🤔 

There are a few outstanding functionality issues as well that you may pick up when testing but they're being looked at separately.

>  Could we instead let the selector take however much horizontal space it needs and allow people to scroll horizontally to see more options?

> Some items contain per bundle... while others using a forward slash /month. Can we standardize these descriptions so that they all use per ...?

Will take a look at these, thanks!

**Comment by coder-karen** [comment]
> Extra spacing in the pricing terms

The per bundle per month info should be on the second line ( I don't see it on the same line).

Interestingly your screenshot looks like you are seeing a default state, without the US$ on it either. That was reported here - https://github.com/Automattic/jetpack-manage/issues/274 - I don't seem to be able to replicate.
So once I can replicate it seems that initial state needs changing, and that should address the spacing issue here (will add a note to that issue).

The Per vs / fix is added and the license selector styling too, ready for review again, thanks!

File: client/jetpack-cloud/sections/manage/pricing/pricing-license-selector/style.scss [hunk 1]
```diff
@@ -15,16 +15,17 @@
 		border-radius: 8px; /* stylelint-disable-line scales/radii */
 		width: 682px;
 		height: 30px;
-		overflow: hidden;
 		margin: 20px auto;
 		padding: 1px 2px;
+		flex-wrap: nowrap;
+		overflow: auto;
```
Comment (gogdzl) [inline client/jetpack-cloud/sections/manage/pricing/pricing-license-selector/style.scss:21]: Tested in two mobile devices, and looks nice!

**Review by tbradsha — APPROVED** [review]
Reviewed the recent commits, and things still look good!

File: client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx [hunk 2]
```diff
@@ -115,6 +117,8 @@ const Paid: React.FC< OwnProps > = ( props ) => {
 		stripZeros: true,
 	} );
 
+	//const customBillingDetails = customTimeFrameBillingTerms ?
```
Comment (kallehauge) [inline client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx]: Looks like a leftover 😄

Comment (coder-karen) [reply client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx]: Good catch!

**Comment by coder-karen** [comment]
> The per bundle per month info should be on the second line ( I don't see it on the same line).

Noting that I worked on fixing this with @keoshi in Slack - p1706869811443229-slack-C02C312V9 - and it seems resolved now with the last commit. The only outstanding question is also on P2 - pf36In-DL-p2#comment-1327- and not related to design but instead extension / add on behaviour, so will follow up separately as needed.

**Pull request merged**